modules/installation-initializing.adoc

```diff
@@ -131,9 +142,7 @@ endif::gcp[]
 link:https://cloud.redhat.com/openshift/install[OpenShift Infrastructure Providers] page.
 
 . Modify the `install-config.yaml` file. You can find more information about
-the available parameters in the *Installation configuration parameters* section
-and in the
-link:https://godoc.org/github.com/openshift/installer/pkg/types#InstallConfig[Go documentation].
+the available parameters in the *Installation configuration parameters* section.
```

Comment:
I don't think bold is appropriate here, that's used pretty much for GUI items. Suggest italics instead.

Reply:
Italics is supposed to be for emphasis on first use of terms, and bold can also be used sparingly for generic emphasis. https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference